modules/apps/frontend-js/frontend-js-components-web/test/treeview/Treeview.js

```diff
@@ -276,4 +276,57 @@ describe('Treeview', () => {
 			);
 		});
 	});
+
+	describe('Treeview with filter prop', () => {
+		it('when a string is passed it compares it with the node names ignoring case', () => {
+			const nodes = [
+				{
+					icon: 'cog',
+					id: '1',
+					name: 'Sandro',
+				},
+				{
+					icon: 'cog',
+					id: '2',
+					name: 'sandro polo',
+				},
+				{
+					icon: 'cog',
+					id: '3',
+					name: 'Pablo',
+				},
+			];
+
+			const {getByText} = render(
+				<Treeview filter={'Sandro'} nodes={nodes} />
+			);
+
+			expect(getByText('Sandro')).toBeInTheDocument();
+			expect(getByText('sandro polo')).toBeInTheDocument();
+		});
+
+		it('uses custom filter function if passed', () => {
```

Comment:
Now this is a verb!!

![yourebreathtaking](https://user-images.githubusercontent.com/905006/98222048-a4f07980-1f50-11eb-8984-7c9ac548c61e.gif)